evm-contracts/src/v0.6/Flags.sol

```diff
@@ -149,4 +159,13 @@ contract Flags is FlagsInterface, SimpleReadAccessController {
       raisingAccessController.hasAccess(msg.sender, msg.data);
   }
 
+  function tryToRaiseFlag(address subject)
+    private
+  {
+    if (!flags[subject]) {
+      flags[subject] = true;
```

Comment:
Can this potentially be a source of concurrency issues, particularly data races? I don't know very well how solidity contracts work.

Reply:
everything is executed in strict sequence, there is no concurrency